src/inputs/labeled-inputs.scss

```diff
@@ -169,6 +167,39 @@
   }
 }
 
+/// When the label wraps around the input.
+@mixin iui-input-container-label {
+  @include iui-input-label-styling;
+  grid-area: label;
+  align-self: center;
+}
+
+/// When the label does not wrap around the input.
+@mixin iui-input-label {
+  @include iui-input-label-styling;
+  width: fit-content;
+  cursor: default;
+
+  @at-root {
+    label#{&} {
+      cursor: pointer;
+
+      &.iui-disabled {
+        cursor: not-allowed;
+      }
+    }
+  }
+
+  &.iui-inline {
+    margin-bottom: 0;
+    margin-right: $iui-m;
+
+    &.iui-required {
+      margin-right: $iui-xs * 1.5;
+    }
+  }
```

Comment:
You might be able to remove these styles and reuse the `iui-input-label-inline` mixin